### PR TITLE
QA: Color token roles

### PR DESCRIPTION
**Pull request by aagonzales**
Updates from QA of color token roles. This work was not previously done pre launch due to time constraints.

### CODE REVIEW

**Review by jnm2377 — APPROVED** [review]
Looks good, just left 1 suggestion where you forgot a closing parens 👍🏽

File: src/data/guidelines/color-tokens.js [hunk 1]
```diff
@@ -1786,7 +1785,7 @@ const colorTokens = {
       },
     },
     '$layer-selected-03': {
-      role: ['$layer-02 selected'],
+      role: ['Selected color for $layer-02'],
```
Comment (laurenmrice) [inline src/data/guidelines/color-tokens.js:1788]:
```suggestion
      role: ['Selected color for $layer-03'],
```

File: src/data/guidelines/color-tokens.js [hunk 2]
```diff
@@ -1870,7 +1869,7 @@ const colorTokens = {
       },
     },
     '$layer-selected-disabled': {
-      role: ['Disabled selected layer'],
+      role: ['Disabled color selected layers'],
```
Comment (laurenmrice) [inline src/data/guidelines/color-tokens.js:1872]:
```suggestion
      role: ['Disabled color for selected layers'],
```

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks great! Just saw these two things^

File: src/data/guidelines/color-tokens.js [hunk 3]
```diff
@@ -1765,7 +1764,7 @@ const colorTokens = {
       },
     },
     '$layer-selected-02': {
-      role: ['$layer-02 selected'],
+      role: ['Selected colro for $layer-02'],
```
Comment (laurenmrice) [inline src/data/guidelines/color-tokens.js:1767]:
```suggestion
      role: ['Selected color for $layer-02'],
```

**Review by laurenmrice — APPROVED** [review]
Looks good to me!

**Pull request merged**